masterlist.yaml

```
@@ -9121,6 +9121,11 @@ plugins:
   - name: 'Crossbow Integration - USSEP.esp'
     tag: [ Text ]
 
+  - name: 'OPHybrid.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/104477/'
+	name: 'Curse of the Dragonborn - REDUX'
+         
```

You've used a tab character here, indentation should only be done with spaces. You also accidentally removed the group assignment & added some trailing spaces. GitHub should give you the ability to commit my suggestion in the website without having to do it manually on your own:
```suggestion
        name: 'Curse of the Dragonborn - REDUX'
    group: *earlyLoadersGroup

```

---

It wasn't letting me commit your suggestions, so I did them manually, including this one. It kept saying "This diff has been updated. Refresh and try again." Either way, I replaced the tabs with spaces, and it passed all checks. Do tabs have a special meaning in this programming language or something? Every other language I've used either didn't care, or actively expected tabs to be used.

---

Ah, GH does that sometimes, still not sure why. Re tab characters: they're illegal in YAML, which the masterlists use, due to inconsistent implementation across editors IIRC.